components/common/listingCart/ListingCartPriceInput.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div class="is-flex">
+    <input
+      v-model="model"
+      type="text"
+      class="price-input height-40 theme-background-color shade-border-color has-text-color"
+      placeholder="Price" />
+    <div
+      class="border-top border-right border-bottom shade-border-color px-5 is-flex is-align-items-center">
+      KSM
```

Comment:
How are we going to handle `DOT` price listing?

Reply:
I'm not sure, right now the data being returned from nftEntities is this
<img width="777" alt="image" src="https://github.com/kodadot/nft-gallery/assets/3810177/9a0d951e-3e70-4ba9-9af9-564529e4d6b4">

Please let me know how I can determine if it is KSM or DOT
Thanks

Reply:
- KSM is on BSX/SNEK/RMRK/AHK chain
- DOT is on AHP chain
- we might integrate some eth (soon?)

I think you'll need to use `chain` from composable